Foosball/FoosballWebsite/app/shared/components/chat.component.ts

```diff
@@ -10,7 +10,6 @@ import { FeedService } from '../services/feed.service';
 export class ChatComponent implements OnInit {
 
     @Input() matches: Match[];
-    @Input() connection: string;
```

Comment:
why is this deleted?

Reply:
it only displays user connection string which is unnecesary